src/main/resources/themes/default/style.css

```diff
@@ -581,6 +581,28 @@
   -fx-background-color: linear-gradient(to left, transparent, -fx-accent);
 }
 
+/***************** Large Map Preview *****************/
+.map-preview-large{
```

Comment:
The general consensus among social scientists is that broken window theory is either wrong or, at best, unproven. :-)
http://www.newrepublic.com/blog/timothy-noah/101318/broken-windows-revisited